package.json

```diff
@@ -2,13 +2,13 @@
   "name": "react-frau-intl",
   "version": "0.0.3",
   "description": "A simple React mixin for getting localized text in components.",
-  "main": "src/react-frau-intl.js",
+  "main": "src/index.js",
   "scripts": {
     "lint": "npm run lint:src && npm run lint:test",
     "lint:src": "jshint src",
     "lint:test": "jshint test",
     "test:unit": "istanbul cover ./node_modules/jasmine-node/bin/jasmine-node -- test --matchall",
-    "test": "npm run lint && npm run test:unit",
+    "test": "npm run test:unit",
```

Comment:
FYI, if you want to keep the linting goodness, there's an `esnext` flag you can use